types/comtac/CM-3/lifecycle.schema.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$id": "https://akenza.io/comtac/CM-3.lifecycle.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Comtac CM-3",
+  "type": "object",
+  "properties": {
+    "tempPt100": {
```

Comment:
this should not be in here -> move to default?

Reply:
These are only flags